src/CMakeLists.txt

```diff
@@ -45,9 +45,27 @@ endif()
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
 
+target_include_directories(cucumber-cpp-nomain PUBLIC
+    $<BUILD_INTERFACE:${CUKE_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>
+    PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
The current source directory isn't necessary as include path so don't add it. Same below